src/components/tooltip/TooltipDirective.js

```diff
@@ -262,13 +263,15 @@ goog.require('ga_topic_service');
                 }, function(enabled) {
                   if (scope.isActive) {
                     if (enabled) {
-                      deregMapEvents();
+                      deregMapEvents = deregMapEvents();
                       deregGlobeEvents = registerGlobeEvents(scope,
                           findFeatures);
                     } else {
                       deregGlobeEvents();
-                      deregMapEvents = registerMapEvents(scope, findFeatures,
-                          gaBrowserSniffer.mobile);
+                      if (deregMapEvents === angular.noop) {
```

Comment:
you mean pass the callback in registerMapEvents and if !== to angular.noop return the function passed initially. yeah i think it's nice. I can adapt.